src/natcap/invest/coastal_blue_carbon/coastal_blue_carbon.py

```diff
@@ -2259,4 +2262,26 @@ def validate(args, limit_to=None):
                         analysis_year=args['analysis_year'],
                         latest_year=max(snapshots.keys()))))
 
+    # check for invalid options in the translation table
+    if ("landcover_transitions_table" not in invalid_keys and
+            "landcover_transitions_table" in sufficient_keys):
+        transitions_spec = MODEL_SPEC['args']['landcover_transitions_table']
+        transition_options = list(
+            transitions_spec['columns']['[LULC CODE]']['options'].keys())
+        # lowercase options since utils call will lowercase table values
+        transition_options = [x.lower() for x in transition_options]
+        transitions_df = utils.read_csv_to_dataframe(
+            args['landcover_transitions_table'], transitions_spec)
+        transitions_mask = ~transitions_df.isin(transition_options) & ~transitions_df.isna()
+        if transitions_mask.any(axis=None):
+            transition_numpy_mask = transitions_mask.values
+            transition_numpy_values = transitions_df.to_numpy()
+            bad_transition_values = list(
+                numpy.unique(transition_numpy_values[transition_numpy_mask]))
+            validation_warnings.append((
+                ['landcover_transitions_table'],
+                INVALID_TRANSITION_VALUES_MSG.format(
+                    model_transitions=(transition_options),
+                    transition_values=bad_transition_values)))
+
```

Comment:
This looks like a good check to me!  Since it is a new custom bit of validation, would it be worth adding a test for this?

Reply:
Done in https://github.com/natcap/invest/pull/1365/commits/1aa4f4cca0cd8196216ab9d0fe03e7048761d4e7